frontend/views/utils/translations.js

```diff
@@ -65,6 +65,15 @@ export default function L (
   return template(key, args)
 }
 
+export function LError (error) {
+  return {
+    reportError: L('Please {a_}report to us{_a} the error.', {
```

Comment:
Please:

1. Add back `error` with `error.message`
2. Make this read `{a_}See error logs.{_a}`